backend/apps/ai/common/extractors/chapter.py

```diff
@@ -0,0 +1,80 @@
+"""Content extractor for Chapter."""
+
+from apps.ai.common.constants import DELIMITER
+
+
+def extract_chapter_content(chapter) -> tuple[str, str]:
+    """Extract structured content from chapter data.
+
+    Args:
+        chapter: Chapter instance
+
+    Returns:
+        tuple[str, str]: (prose_content, metadata_content)
+
+    """
+    prose_parts = []
+    metadata_parts = []
+
+    if chapter.description:
+        prose_parts.append(f"Description: {chapter.description}")
+
+    if chapter.summary:
+        prose_parts.append(f"Summary: {chapter.summary}")
+
+    if chapter.owasp_repository:
+        repo = chapter.owasp_repository
+        if repo.description:
+            prose_parts.append(f"Repository Description: {repo.description}")
+        if repo.topics and hasattr(repo.topics, "__iter__") and not isinstance(repo.topics, str):
```

Comment:
Why is this needed? Do we somehow store different type content in chapter topics?